File: 392-is-subsequence/392-is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        count = 0
        check = 0 
        for i in s:
            if i in t:
                count = t.find(i)+1
                t = t[count:]
            else:
                check = 1
                break 
        if check == 0 :
            return True
        else:
            return False

File: 392-is-subsequence/392-is-subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        for (char c:s){
            string s1 = t;
            for (int i = 0 ; i < t.length();i++){
                if (c==t[i]){
                    t=t.substr(i+1,t.length());
                    break;
                }
            }
            if (s1 == t){
                return false;
            }
        }
        return true;
    }
};